components/Proposals/JupSwapCard.tsx

```diff
@@ -140,38 +129,31 @@ export function JupSwapCard() {
       <Divider />
       <Title order={5}>Swap</Title>
       <Group justify="space-between">
-        <Image
-          priority
-          src={poweredByJup}
-          alt="Powered By Jupiter"
-        />
-        {(balance && balance.amount !== undefined) &&
+        <Image priority src={poweredByJup} alt="Powered By Jupiter" />
+        {balance && balance.amount !== undefined && (
```

Comment:
only thing to worry about here is if `balance.amount === null` then the below code might bomb. We might want to consider doing `balance && balance.amount && (<ReactBlah />)`